pxr/usdImaging/lib/usdImagingGL/drawModeAdapter.cpp

```diff
@@ -1092,11 +1092,23 @@ UsdImagingGLDrawModeAdapter::_ComputeExtent(UsdPrim const& prim) const
 {
     HD_TRACE_FUNCTION();
     HF_MALLOC_TAG_FUNCTION();
-
-    TfTokenVector purposes = { UsdGeomTokens->default_, UsdGeomTokens->proxy,
-                               UsdGeomTokens->render };
-    UsdGeomBBoxCache bboxCache(UsdTimeCode::EarliestTime(), purposes, true);
-    return bboxCache.ComputeUntransformedBound(prim).ComputeAlignedBox();
+    if (prim.IsLoaded()) {
+        TfTokenVector purposes = { UsdGeomTokens->default_,
+            UsdGeomTokens->proxy, UsdGeomTokens->render };
+        UsdGeomBBoxCache bboxCache(
+            UsdTimeCode::EarliestTime(), purposes, true);
+        return bboxCache.ComputeUntransformedBound(prim).ComputeAlignedBox();
+    } else {
+        GfRange3d extent;
```

Comment:
As I alluded to in the comment for the header file, you can check here to see if prim.IsA<UsdGeomBoundable>(), and if so, see if it has an authored extent you can use (this is probably an uncommon case - at least it would be for us, but maybe it's worth it?)